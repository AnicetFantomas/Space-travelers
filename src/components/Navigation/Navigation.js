import React from 'react';
import { NavLink } from 'react-router-dom';
import './Navigation.css';

const Navigation = () => (
  <div className="Navigation-bar">
    <h1 className="nav-title">KanBan Project</h1>
    <div className="navLinks">
      <NavLink exact to="/" activeClassName="nav-active" className="link-el">Rockets</NavLink>
      <NavLink exact to="/Missions" activeClassName="nav-active" className="link-el">Missions</NavLink>
      <NavLink exact to="/MyProfile" activeClassName="nav-active" className="link-el">My Profile</NavLink>
    </div>
  </div>
);

export default Navigation;
