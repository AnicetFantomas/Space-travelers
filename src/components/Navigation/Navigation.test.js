import React from 'react';
import renderer from 'react-test-renderer';
import { HashRouter as Router } from 'react-router-dom';
import { Provider } from 'react-redux';
import store from '../Navigation/configuration';
import Navigation from './Navigation';

describe('test missions component', () => {
    it('Render missions row', () => {
        const app = renderer
        .create(
            <Provider store={store}>
                <Router>
                    <Navigation />
                </Router>
            </Provider>
        ).toJSON();
        expect(app).toMatchSnapshot()
    })
})