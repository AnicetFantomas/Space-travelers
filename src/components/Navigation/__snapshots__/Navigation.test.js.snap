// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test missions component Render missions row 1`] = `
<div
  className="Navigation-bar"
>
  <h1
    className="nav-title"
  >
    KanBan Project
  </h1>
  <div
    className="navLinks"
  >
    <a
      activeClassName="nav-active"
      aria-current="page"
      className="link-el active"
      exact={true}
      href="#/"
      onClick={[Function]}
    >
      Rockets
    </a>
    <a
      activeClassName="nav-active"
      className="link-el"
      exact={true}
      href="#/Missions"
      onClick={[Function]}
    >
      Missions
    </a>
    <a
      activeClassName="nav-active"
      className="link-el"
      exact={true}
      href="#/MyProfile"
      onClick={[Function]}
    >
      My Profile
    </a>
  </div>
</div>
`;
